src/style/common/modal.less

```diff
@@ -282,10 +282,12 @@
   }
 
   &__header {
-    position: relative;
-    min-height: 24px;
-    padding-top: 12px;
+    position: sticky;
+    top: 0;
+    min-height: 32px;
+    padding: 32px 32px 0 32px;
```

Comment:
padding of 32px on the top and side is close-icon-size + close-icon-margin (16px+16px)